manifests/server/resources.pp

```diff
@@ -13,7 +13,7 @@
   Boolean $manage_flannel              = true,
   K8s::Proxy_method $manage_kube_proxy = $k8s::manage_kube_proxy,
 
-  String[1] $kube_proxy_image                    = 'k8s.gcr.io/kube-proxy',
+  String[1] $kube_proxy_image                    = "${k8s::container_registry}/kube-proxy",
```

Comment:
just a question: why don't you do it like in the other places where you changed the image/registry?

Reply:
Mainly it was just to keep things consistent inside the resources class, but I guess it makes more sense to just add a registry parameter to all of them.